lib/inspec/resources/apache_conf.rb

```diff
@@ -101,12 +101,14 @@ def include_files(params)
       include_files_optional = params["IncludeOptional"] || []
 
       includes = []
-      (include_files + include_files_optional).each do |f|
-        id    = Pathname.new(f).absolute? ? f : File.join(conf_dir, f)
-        files = find_files(id, depth: 1, type: "file")
-        files += find_files(id, depth: 1, type: "link")
+      unless conf_dir.nil?
```

Comment:
How we handle if conf_dir is nil ?

Reply:
Yes I need to discuss the possible solution for this on today's standup hence it is in WIP right now